<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLineEdit" name="usernameLineEdit">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>149</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordLineEdit">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>221</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="loginButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>280</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QLabel" name="usernamelabel">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>113</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string>username</string>
   </property>
  </widget>
  <widget class="QLabel" name="passwordlabel">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>185</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string>password</string>
   </property>
  </widget>
  <widget class="QLabel" name="statusLabel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>190</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>statusLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
